etc/kayobe/kolla/config/prometheus/prometheus.yml.d/60-redfish.yml

```diff
@@ -0,0 +1,25 @@
+{% if seed_redfish_exporter_container_enabled | bool %}
+scrape_configs:
+  - job_name: redfish-exporter-seed
+    metrics_path: /redfish
+    scrape_timeout: 120s
+    scrape_interval: {{ 8 * groups['redfish_exporter_targets'] | length }}s
```

Comment:
Perhaps we'd want a minimum interval here too to avoid excessive scraping in small clouds?

Reply:
What would a sensible minimum be? 60s?

Reply:
I think everyone expects to have data of, at least, this granularity.